packages/core/src/index.ts

```diff
@@ -2,15 +2,25 @@
 /* eslint-disable import/first */
 import { curry2, curry3, Curried2 } from '@most/prelude'
 
+// -----------------------------------------------------------------------
+// Construction
+
 export { empty } from './source/empty'
 export { never } from './source/never'
 export { now } from './source/now'
-export { at } from './source/at'
-
 export { periodic } from './source/periodic'
-
 export { newStream } from './source/newStream'
 
+// -------------------------------------------------------
+
+import { at as _at } from './source/at'
+
+interface At {
+  <A>(time: Time, value: A): Stream<A>
+  <A>(time: Time): (value: A) => Stream<A>
```

Comment:
Oh jeez, I'm sorry.  It didn't occur to me in https://github.com/mostjs/core/issues/676 that this might be a breaking change for _JavaScript_ users.  I just noticed it now that I see these type signatures.

In the non-curried version, `at`'s TS type requires the second parameter, i.e. the value, but a JS caller _could_ omit it, e.g. `at(time)`, and it would return a `Stream`.

By currying it, the JS behavior will change: instead of `at(time)` returning `Stream`, it will return a function `(value) => Stream` 😕 

We'd need to release it as 2.0.0 to signal the breaking change.  It's not a blocker, though, since I don't really care about the "marketing value" of version numbers.  I figured it was worth calling out, though.

That might be reason enough to separate this into 2 PRs: one for the non-breaking changes, and one for the breaking change to `at`.

Thoughts on all of that?

Reply:
Ok I understand, I too missed that this is a breaking change.

I think, currying `at` is a too small change that it alone not justifies releasing a "2.0". 
I reverted that commit, so that *this PR* now only deals with the non-breaking changes to the documentation.

I could author another PR for `at`, but rather I'd somehow bookmark the commit as "2.0-planned" to include later if the real need for a version "2.0" should arise. 

Perhaps you could mark the issue #676 with a label like "2.0"?

Reply:
I've updated the signature for at in the documentation to reflect it's signature as uncurried binary and unary function.

Reply:
> Perhaps you could mark the issue https://github.com/mostjs/core/issues/676 with a label like "2.0"?

Great idea!  Done.